websites/E/e-typing/presence.ts

```diff
@@ -0,0 +1,116 @@
+import { ActivityType } from 'premid'
+
+const presence = new Presence({
+  clientId: '1343107487477399592',
+})
+
+const romaLargeImageKey = 'https://i.imgur.com/FbaFi9M.png'
+const englishLargeImageKey = 'https://i.imgur.com/55iKGjg.png'
+const kanaLargeImageKey = 'https://i.imgur.com/gJwNzqB.png'
+const browsingTimestamp = Math.floor(Date.now() / 1000)
+
+presence.on('UpdateData', async () => {
+  const privacy = await presence.getSetting<boolean>('privacy')
+  if (privacy) {
+    return presence.clearActivity()
+  }
```

Comment:
Not very useful - duplicates extension disable function. Remove setting or change to show generic messages